config/ml/repos/victordibia-handtrack.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": 1.0,
+    "source": "github",
+    "owner": "victordibia",
+    "repo": "handtrack.js/",
```

Comment:
This repo is not valid, I think it's the slash?

Reply:
yup